### PR TITLE
PTAPP-317:  Updates for dart 3.

### PR DESCRIPTION
**Pull request by cah-aamir**

### CODE REVIEW

File: lib/src/log_console.dart [hunk 1]
```diff
@@ -269,7 +273,7 @@ class _LogConsoleState extends State<LogConsole> {
             items: [
               DropdownMenuItem(
                 child: Text("VERBOSE"),
```
Comment (cah-ken) [inline lib/src/log_console.dart]: Should the text here change to match the value?

File: lib/src/log_console.dart [hunk 2]
```diff
@@ -289,7 +293,7 @@ class _LogConsoleState extends State<LogConsole> {
               ),
               DropdownMenuItem(
                 child: Text("WTF"),
```
Comment (cah-ken) [inline lib/src/log_console.dart]: Should the text here change to match the value?

**Review by cah-ken — APPROVED** [review]

File: lib/src/log_console.dart (hunk 1 above)
Comment (cah-aamir) [reply lib/src/log_console.dart]: Fixed

File: lib/src/log_console.dart (hunk 2 above)
Comment (cah-aamir) [reply lib/src/log_console.dart]: Fixed

**Review by cah-ken — APPROVED** [review]

**Pull request merged**